src/main/groovy/org/pih/warehouse/api/PartialReceipt.groovy

```diff
@@ -41,6 +41,10 @@ class PartialReceipt {
         return partialReceiptItems
     }
 
+    Boolean isShipmentOrderBased() {
+        return shipment.orders.size() > 0
```

Comment:
wasn't it supposed to be only applied to the shipment from PO?
if so, this should be changed, because currently it would return true also for returns and putaways.

Reply:
Could you just use `shipment.isFromPurchaseOrder` instead, if it is only for POs?